website/content/docs/operators/deployment/schedulers/yarn.md

```diff
@@ -30,16 +30,29 @@ containers are allocated for each topology.
 
 ### Configuring the Heron client classpath
 
-1. Command `hadoop classpath` provides a list of jars needed to submit a hadoop job. Copy all jars
-to `HERON_INSTALL_DIR/lib/scheduler`.
-  * Do not copy commons-cli jar if it is older than version 1.3.1.
-1. Create a jar containing core-site.xml and yarn-site.xml. Add this jar to
-`HERON_INSTALL_DIR/lib/scheduler` too.
-
-> The YARN scheduler needs Hadoop and YARN jars in the classpath. Heron currently does not allow
-> customizing client classpath. Hadoop and YARN jars are version dependent and should not be
-> bundled with the Heron distribution. So, this manual step is needed till this Heron issue
-> ([271] (https://github.com/twitter/heron/issues/271)) is fixed.
+**Under 0.14.2 version (including 0.14.2)**
```

Comment:
@mycFelix - should we need this since we have released 0.14.3 already and 0.14.4 is going to be released this week or early next week?